data/mcscript/submissions/submissions_metadata.jsonl

```diff
@@ -1 +1 @@
-{"@context":"http://schema.org/","@id":"MCScript","@type":"Submission","name":"mcscript-trian","description":"This is the TriAN model that was used to evaluate and refine the MCScript dataset. It achieves an accuracy of 81.9% after 1.5 epochs. https://github.com/yuantiku/commonsense-rc","dateCreated":"10-29-2020T20:56:30Z","isBasedOn":"MCScript","contributor":{"@type":"Person","name":"Yuanfudao"},"contentRating":[{"@type":"TestScore","isBasedOn":"MCScript/test","name":"Test Score","value":"0.4983"},{"@type":"DevScore","isBasedOn":"MCScript/dev","name":"Dev Score","value":"0.8228"}],"resultOf":{"@type":"SoftwareApplication","startTime":"10-29-2020T20:56:30Z","endTime":"10-29-2020T21:15:11Z","url":"https://github.com/yuantiku/commonsense-rc"}}
+{"@context":"http://schema.org/","@id":"MCScript","@type":"Submission","name":"mcscript-trian","description":"This is the TriAN model that was used to evaluate and refine the MCScript dataset. It achieves an accuracy of 81.9% after 1.5 epochs. https://github.com/yuantiku/commonsense-rc","dateCreated":"10-29-2020T20:56:30Z","isBasedOn":"MCScript","contributor":[{"@type":"Person","name":"Yuanfudao"}],"contentRating":[{"@type":"TestScore","isBasedOn":"MCScript/test","name":"Test Score","value":"0.4983"},{"@type":"DevScore","isBasedOn":"MCScript/dev","name":"Dev Score","value":"0.8228"}],"resultOf":{"@type":"SoftwareApplication","startTime":"10-29-2020T20:56:30Z","endTime":"10-29-2020T21:15:11Z","url":"https://github.com/yuantiku/commonsense-rc"}}

```

Comment:
Not sure why all of these data/ files are changed? Since they're only a single line the diff is not obvious.

Reply:
I changed the "Contributor" entry to be a list, rather than just a single entry. Minor change